src/linear_algebra/bilinear_form.lean

```diff
@@ -6,9 +6,6 @@ Authors: Andreas Swerdlow, Kexing Ying
 
 import linear_algebra.dual
 import linear_algebra.matrix.basis
```

Comment:
Is this import used any longer?

Reply:
yes, otherwise it fails to find an instance `finite_dimensional K (V →ₗ[K] K)` for `finite_dimensional K V` on line 1128

Reply:
and I didn't want to look how easy it is to fix that because I think that just doing the c&p in this PR makes it easier to review.

Reply:
```suggestion
import linear_algebra.matrix.to_lin
```
is where that instance comes from.